File: code/TemplateStudioForUWP/Templates/_comp/P/SampleData.ImageGallery.xUnit/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.cs
//{[{
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.XUnit
{
    public class Tests
    {
        //^^
        //{[{

        // Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Fact]
        public async Task EnsureSampleDataServiceReturnsImageGalleryDataAsync()
        {
            var dataService = new SampleDataService();
            var actual = await dataService.GetImageGalleryDataAsync("ms-appx:///Assets");

            Assert.NotEmpty(actual);
        }
        //}]}
    }
}

File: code/TemplateStudioForUWP/Templates/Uwp/_comp/_shared/Test.SampleData.Grids.xUnit._VB/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.vb
'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Fact>
    Public Async Function EnsureSampleDataServiceReturnsGridDataAsync() As Task
        Dim actual = Await SampleDataService.GetGridDataAsync()

        Assert.NotEmpty(actual)
    End Function
    '}]}
End Class
